eng/pipelines/templates/stages/vmr-build.yml

```diff
@@ -311,6 +311,7 @@ stages:
         container: ${{ variables.ubuntu2204Container }}
         targetOS: linux
         targetArchitecture: x64
+        useDevVersions: true # Use dev versions for CI validation of the experience. If we decide to ship assets from this leg, then we should remove this option.
```

Comment:
can you please condition this job on running in public, that should avoid any confusion and prevent us from accidentally consuming artifacts from it.

also change the buildname to have "dev versions" in it or something so the GitHub status reflects it

Reply:
actually, I just realized that this is the only Linux job we run in public installer jobs (apart from short stacks), are we fine with this changing to -dev versions or should we add another job?